src/seaport.ts

```diff
@@ -2881,7 +2895,7 @@ export class OpenSeaPort {
       ? makeBigNumber(this.web3.toWei(priceDiff, 'ether')).round()
       : WyvernProtocol.toBaseUnitAmount(makeBigNumber(priceDiff), token.decimals)
 
-    return { basePrice, extra, paymentToken }
+    return { basePrice, extra, paymentToken, englishAuctionReservePrice }
```

Comment:
should have caught this earlier but we should have basePrice and englishAuctionReservePrice use the same unit system, e.g. both should be converted to the base unit amount